tests/lib/rules/no-plusplus.js

```diff
@@ -81,6 +85,61 @@ ruleTester.run("no-plusplus", rule, {
                 },
                 type: "UpdateExpression"
             }]
+        },
+        {
+            code: "for (i++;;);",
+            options: [{ allowForLoopAfterthoughts: true }],
+            errors: [{
+                messageId: "unexpectedUnaryOp",
+                data: {
+                    operator: "++"
+                },
+                type: "UpdateExpression"
+            }]
+        },
+        {
+            code: "for (;--i;);",
+            options: [{ allowForLoopAfterthoughts: true }],
+            errors: [{
+                messageId: "unexpectedUnaryOp",
+                data: {
+                    operator: "--"
+                },
+                type: "UpdateExpression"
+            }]
+        },
+        {
+            code: "for (;;) ++i;",
+            options: [{ allowForLoopAfterthoughts: true }],
+            errors: [{
+                messageId: "unexpectedUnaryOp",
+                data: {
+                    operator: "++"
+                },
+                type: "UpdateExpression"
+            }]
+        },
+        {
+            code: "for (;; i = j++);",
+            options: [{ allowForLoopAfterthoughts: true }],
+            errors: [{
+                messageId: "unexpectedUnaryOp",
+                data: {
+                    operator: "++"
+                },
+                type: "UpdateExpression"
+            }]
+        },
+        {
+            code: "for (;; i++, f(--j));",
```

Comment:
non-blocking questions for understanding the proper behavior of this option.

It seems the option`{allowForLoopAfterthoughts: true}` changed to allow `update expression`s in `sequence expression`, which has exactly one depth from `for-loop update`.

so the case like ` f(--j)` is warned.

And can I understand the below case(nested sequence expression ) in the same manner?

```js
for (;; (++a, (++b, ++c) ) {}  // it's warned in the PR version. no-plusplus for `b` and `c`
```

Reply:
Didn't think of that. It's done now, thanks!